.github/workflows/make-check.yml

```diff
@@ -27,3 +27,8 @@ jobs:
           go-version: ${{ matrix.go }}
       - name: Build
         run: make check
+      - name: Upload artifacts
```

Comment:
This might fit better in make-rpm.yml, for other platforms we ship installers, not raw binaries.

Reply:
I can try it to `make-rpm` action.

Reply:
@cfergeau I prefer binary instead of rpm if we need to share someone who is using non-rpm base systems.

Reply:
Let's keep it in `make check` then even if none of the places are great for generating/exporting a `crc` binary artifact.

Reply:
@cfergeau `make check` run on each runner not separate like installer so have to check if we can have a condition that only use linux binary for linux runner system.